actors/market/src/types.rs

```diff
@@ -216,3 +217,13 @@ pub struct GetDealActivationReturn {
     /// Epoch at which the deal was terminated abnormally, or -1.
     pub terminated: ChainEpoch,
 }
+
+// Interface market clients can implement to receive notifications from builtin market
+pub const MARKET_NOTIFY_DEAL: u64 = frc42_dispatch::method_hash!("MarketNotifyDeal");
```

Comment:
```suggestion
pub const MARKET_NOTIFY_DEAL_METHOD: u64 = frc42_dispatch::method_hash!("MarketNotifyDeal");
```

for consistency with other such consts